.github/workflows/test.yml

```diff
@@ -138,12 +138,20 @@ jobs:
           ref: ${{ github.head_ref }}
           submodules: "recursive"
 
+      - name: Preserve the ArkanaKeys (avoid code change in git when refreshing the "sample" project)
```

Comment:
What do you think about moving `(avoid code change in git when refreshing the "sample" project)` to the comment, we should keep the name short and clean 🤔 

```
# It avoids code change in git when refreshing the "sample" project
- name: Preserve the ArkanaKeys
```